src/sentry/static/sentry/app/views/releases/list/organizationReleases/index.jsx

```diff
@@ -118,12 +117,12 @@ class OrganizationReleases extends AsyncView {
   renderReleaseProgress() {
     const {organization, selection} = this.props;
     const allAccessibleProjects = organization.projects.filter(
-      project => project.isMember
+      project => project.hasAccess
```

Comment:
We use `project.isMember` a lot, are there other places we need to replace it?

Reply:
There may be more, will follow up